docs/source/conf.py

```diff
@@ -37,14 +37,10 @@
 import warnings
 from unittest import mock
 
+import datafusion
 import pyarrow
 
-
-sys.path.extend([
-    os.path.join(os.path.dirname(__file__),
-                 '..', '../..')
-
-])
+sys.path.extend([os.path.join(os.path.dirname(__file__), "..", "../..")])
```

Comment:
Most of this file is auto-generated. By making style changes we will make it more difficult to diff the current file against a pristine file.